css/properties/flex-grow.json

```diff
@@ -76,7 +76,6 @@
               }
             ],
             "ie": {
-              "alternative_name": "-ms-flex-positive",
               "version_added": "11"
```

Comment:
I think this should be changed to:
```json
"ie": [
  {
    "version_added": "11"
  },
  {
    "version_added": "11",
    "alternative_name": "-ms-flex-positive"
  }
]
```

Reply:
The two objects here represent our current way to signal that this is available under both, the standard name and the alternative_name. The versions could differ, but seems both were added in the same version.